snowbox.js.org

yuxizhe opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But your page is having some path issues now. Please fix them to give me a chance to have a look.
I'll merge after we have "contact". Please keep this PR open.

yuxizhe commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But your page is having some path issues now. Please fix them to give me a chance to have a look. I'll merge after we have "contact". Please keep this PR open.

@indus Thanks! I have fixed them. Its working now https://snowbox.js.org/

indus commented:
Welcome https://snowbox.js.org
